string ACTION_NAME=/oni_vicon_player/Play

uint32 starting_frame
---
---
bool playing
float64 current_time
uint32 current_vicon_frame
uint32 current_depth_sensor_frame
